creusot/tests/should_succeed/hashmap.coma

```diff
@@ -933,7 +921,11 @@ module M_hashmap__qyi7664122466964245986__add
     | bb15 = s0 [ s0 =  [ &_47 <- List'0.C_Cons _48 _52 ] s1 | s1 = bb16 ] 
     | bb16 = bb17
     | bb17 = s0
-      [ s0 = {[@expl:type invariant] inv'1 l.current} s1
+      [ s0 = {[@expl:type invariant] match l with
+          | {current = inv_asrt'0} -> inv'1 inv_asrt'0
+          | _ -> true
+          end}
```

Comment:
the matches generated here are worse than the prior code, but fixing this would be a lot more sophisticated and I would rather defer that.